.travis.yml

```
@@ -1,130 +1,166 @@
+# Runs Test scripts; Linting; Deployment on Travis-CI
 language: python
 python:
-  # We don't actually use the Travis Python, but this keeps it organized.
   #- "2.7"    # not tested anymore starting from 0.9.19
-  - "3.8"
+  - "3.8" #end of life, Novembre 2024, https://devguide.python.org/versions/
+  - "3.10"
+  - "3.11"
```

@minouHub removing 3.8 to save some calculation time

```suggestion
  - "3.10"  # has vaex support
  - "3.11"  # has no vaex support (June 24)
```

---

ok, will be applied the next commit